04week/loop.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+
+//this part is from the arrays practice but is needed for this loops practice
+const cars = ["Ford", "Tesla", "Chevy", "Jeep"];
+console.log(cars);
+
+const moreCars = ["Nissan", "Audi", "Volkswagon", "Honda"];
+console.log(moreCars.length);
+
+const totalCars = cars.concat(moreCars);
+console.log(totalCars.indexOf("Honda"));
+console.log(totalCars.lastIndexOf("Ford"));
+
+//commenting the two below out for now since totalCars is used twice
+//const stringOfCars = totalCars.join();
+//const totalCars = stringOfCars.split();
+
+const carsInReverse = totalCars.reverse();
+
+// use a for loop to console.log each item in the carsInReverse array
+var i;
+for (carsInReverse[i] = 0; carsInReverse[i] < carsInReverse.length - 1; i++) {
+  console.log(carsInReverse[i]);
+};
+
+
+// create an object called persons with the data given on assignment and use a "for...in" loop to console log each key, then to console.log the value associated with the birthDate key
+const persons = {
+  firstName = "Jane",
+  lastName = "Doe",
+  birthDate = "Jan. 5, 1925",
+  gender = "Female"
+}
+
+var x;
+for (var key in persons) {
+  console.log(x)
+}
+
```

Comment:
don't use var, and you need to do second part of question to console the key associated with birthDate